scripts/ci/kg_drift_gate.py

```
@@ -0,0 +1,126 @@
+    """
+    Return the Git commit SHA for HEAD in the given repository root.
+    
+    Parameters:
+        root (Path): Filesystem path of the repository in which to run `git rev-parse HEAD`.
+    
+    Returns:
+        sha (str): The trimmed commit SHA of HEAD.
+    
+    """
+    result = subprocess.run(
+        ["git", "rev-parse", "HEAD"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)